server.js

```diff
@@ -24,9 +13,27 @@ app.use(bodyParser.json())
 
 // Start defining your routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('We are advocates of avocados. /avocados for full list. /id/:id for specific id, /regions/:region for specific region')
+})
+
+app.get('/avocados', (req, res) => {
+  res.json(avocadoSalesData)
+})
+
+app.get('/avocados/id/:id', (request, response) => {
+  const { id } = request.params
+  const specificID = avocadoSalesData.find((avocado) => avocado.id === +id)
+  response.json(specificID)
 })
 
+app.get('/avocados/regions/:region', (request, response) => {
+  const { region } = request.params
+
+  let specificRegion = avocadoSalesData.filter((avocado) => avocado.region === region)
+  response.json(specificRegion)
+})
```

Comment:
Nice with some filtering! Endpoints should be named after what they return and not how they are filtered so since you're returning avocados and not regions, this could instead be a query param under your main endpoint:
`/avocados?region=albany`